tests/conftest.py

```diff
@@ -1,8 +1,20 @@
 import pytest
+import responses
 
 from web.app import app
 
 
 @pytest.fixture
 def client():
     return app.test_client()
+
+
+@pytest.fixture
+def knowledge_graph_stub():
```

Comment:
It'd be useful to add some test cases over successful knowledge graph responses, as well as this failure case response.